code/datums/uplink_item.dm

```diff
@@ -1126,22 +1126,36 @@ var/list/uplink_items = list()
 	name = "Chemical Dart Gun"
 	desc = "A staple in vox weaponry. This dart gun starts loaded with darts containing sleep toxin and chloral hydrate. The beaker inside can be swapped out to create your own deadly mixes."
 	item = /obj/item/weapon/gun/dartgun/vox/raider
-	cost = 16
-	jobs_exclusive = list("Trader")
+	cost = 20
+	discounted_cost = 16
+	jobs_exclusive = list("Trader","Vox","Skeletal Vox")
+	jobs_with_discount = list("Trader")
 
 /datum/uplink_item/jobspecific/trader/dart_cartridge
 	name = "Dart Cartridge"
 	desc = "A spare cartridge to refill your dart gun."
 	item = /obj/item/weapon/dart_cartridge
-	cost = 2
-	jobs_exclusive = list("Trader")
+	cost = 6
+	discounted_cost = 2
+	jobs_exclusive = list("Trader","Vox","Skeletal Vox")
+	jobs_with_discount = list("Trader")
 
 /datum/uplink_item/jobspecific/trader/cratesender
 	name = "Modified Crate Sender"
 	desc = "A modified salvage crate sender that has been modified to bypass the security protocols, allowing it to teleport crates from onboard the station and allowing it to teleport crates to random destinations. Comes with a cargo telepad you can send your stolen goods to."
 	item = /obj/item/weapon/storage/box/syndie_kit/cratesender
+	cost = 10
+	discounted_cost = 6
+	jobs_exclusive = list("Trader","Vox","Skeletal Vox")
+	jobs_with_discount = list("Trader","Cargo Technician","Quartermaster")
+
+/datum/uplink_item/jobspecific/cannedmatter
+	category = "Skrell Specials"
+	name = "Canned Compressed Matter"
+	desc = "For once, the syndicate has it."
```

Comment:
the description needs to be more clear about what it actually does; you can keep the joke at the end of the desc

Reply:
Fixed